Frameworks/AddressBook/ABAddressBookManagerInternal.h

```diff
@@ -17,10 +17,25 @@
 #pragma once
 
 #import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABRecord.h>
 
 @interface _ABAddressBookManager : NSObject
```

Comment:
I guess maybe _ABContactManager to match the windows name then? (I still am of the opinion that 'manager' is a terrible thing to name a class, but that's a bit out of our hands I guess)

---
In reply to: [77095804](https://github.com/Microsoft/WinObjC/pull/935#discussion_r77095804) [](ancestors = 77095804,77094758)